plot.py

```diff
@@ -330,14 +329,14 @@ def plotDataMC(args,plot):
 	if args.ratio:
 
 		ratioPad.RedrawAxis()
-	if not os.path.exists("plots"):
-		os.makedirs("plots")	
+	if not os.path.exists("test"):
```

Comment:
This doesn't look that useful to me, why not keep "plots"?

Reply:
sorry my bad, will change this back to plots.